js/interactive-guides/microprofile-config/playground-callback.js

```diff
@@ -257,6 +258,13 @@ var playground = function(){
                     // Change the background color to always match the card
                     card.css('background-color', configSource.bgcolor);
 
+                    //adding aria-labels to ordinal cards
+                    var cardInfo = card.find('.ordinalCardFileName').html();
+                      if (card.find('.ordinalCardOrdinal').html()){
```

Comment:
We are doing the retrieving the html twice and can check it after the first line.